HandPuzzle/Handpuzzelcode/Handpuzzelcode.ino

```diff
@@ -0,0 +1,23 @@
+int analogPin = A3; // potentiometer wiper (middle terminal) connected to analog pin 3
+                    // outside leads to ground and +5V
+int val = 0;  // variable to store the value read
+
+void setup() {
+  Serial.begin(9600); //  setup serial
+  pinMode(13, OUTPUT); //led
```

Comment:
Gebruik i.p.v. magic number 13 dan hier een variable of define. Geldt ook voor de regel hier onder.